lib/public/css/nav.scss

```diff
@@ -1,50 +1,117 @@
 .nav {
-    padding: 0 3em;
-    display: flex;
+  background-color: $dark-orange;
+  border-bottom: 2px solid $tan;
+  display: flex;
 
-    .nav__logo-wrapper {
```

Comment:
@jdonisvitch based upon what you were saying last week, did you think I was trying to write `nav__logo--wrapper` here?  It's totally inconsequential now, but I was actually trying to write `logo-wrapper` as the BEM element.  Is that the wrong way to write an element name in BEM?  I couldn't figure-out how to handle multiple words correctly after doing a little research, so I used `dash-case`.

Reply:
Not sure if I did this `100% correct`, I suppose in this case, you'd probably just create a wrapper class that would be used in any scenario. Since each wrapper may be different, and I kind of liked the idea I just added a modifier to the nav logo `--wrapper`  rather than a an element `__wrapper`. It may be apples to oranges. Good point tho!